.standard_todo.yml

```diff
@@ -1,6 +1,32 @@
 ---
 ignore:
-  - 'lib/trino/client/*.rb'
```

Comment:
I'm not sure why the target files are expanded.

I think it's enough to add the rules to the current target.

Reply:
revereted